docker/Dockerfile-windows

```diff
@@ -1,6 +1,12 @@
 ARG WINDOWS_VERSION=1809
 #This version confirmed to work for both github and AKS
 FROM ghcr.io/dapr/windows-base:$WINDOWS_VERSION
+
+# Administrator access is required to install certificates.
+USER ContainerAdministrator
```

Comment:
Is there a way for this not to be the default user? I assume most users will not need to install certificates. Maybe we need to publish a new image.

Reply:
This is a build-time setting, right? We can publish two images and select one in pod_patch based on if this environment variable is set, but that won't be very clean. 

Another option is to add another entrypoint logic in a `cmd` file, and based on if admin permissions are needed or not (i.e. if this environment variable is not set), it can runas `ContainerUser` instead of `ContainerAdministrator`. I will explore more on this.

Reply:
Please. I think not forcing Admin user is key.

Reply:
Yes @ItalyPaleAle, I will test these out! Will reply on this thread.

Reply:
With the latest update, we have a logic in-place to use Admin user if `SSL_CERT_DIR` was set using k8s YAML (as suggested by @mukundansundar) in the sidecar injector. This has been verified on both Linux and Windows.

Note, for Linux containers, the container will still contain the `windowsOptions` YAML, but it won't have any effect.
![image](https://user-images.githubusercontent.com/22132944/175647643-d42a5a82-0b7d-4a39-9db3-4594a182e748.png)

Reply:
This works for me, thanks @shubham1172 for the investigation